modules/controller/rpc-controller.js

```diff
@@ -227,40 +179,70 @@ class RpcController {
                     status: 'PENDING',
                 });
                 const handlerId = inserted_object.dataValues.handler_id;
-                res.status(200).send({
+                res.status(202).send({
```

Comment:
I think that we should return 200 as a status for this request. If we are looking from the user perspective this resolve api call is finished, it's purpose is to start resolve process and to return handler id. From the perspective of the resolve/result route it makes sence that it returns 202 status code untill processing is finalized. What do you think?

Reply:
For me it looks good to use 202 because it means: 

''The HyperText Transfer Protocol (HTTP) 202 Accepted response status code indicates that the request has been accepted for processing, but the processing has not been completed; in fact, processing may not have started yet.''

[Reference link](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202)